src/chainConstants.ts

```diff
@@ -32,3 +32,5 @@ export const EWC_DOMAIN_NOTIFER_ADDRESS =
 export const EWC_IDENTITY_MANAGER_ADDRESS =
   '0x2d1569f3a2006d21c0dc60eb13c8557b63ce5a8d';
 export const EWC_ADDRESS_1056 = '0xE29672f34e92b56C9169f9D485fFc8b9A136BCE4';
+export const EWC_RESOLVER_V2_ADDRESS =
+  '0xcf72f16Ab886776232bea2fcf3689761a0b74EfE';
```

Comment:
@JGiter @Harasz I don't think this is correct for EWC: https://explorer.energyweb.org/address/0xcf72f16Ab886776232bea2fcf3689761a0b74EfE/transactions
Am I misunderstanding something?

Reply:
Yeah, you're right. I didn't check it.

Reply:
@JGiter shall we revert this change?

Reply:
My bad. Fixing it in https://github.com/energywebfoundation/iam-contracts/pull/113

Reply:
No worries, thanks